### DISCUSSION TITLE
[nsfw] adam4adam.com: breakage

### DISCUSSION THREAD

**Discussion by SmoochySix4** [uBlockOrigin/uAssets#26094] [General] [answered]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [x] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.adam4adam.com/last-visits`

### Category

breakage

### Description

When using `*$popunder` in a filter/list, clicking the site logo to return to the main page after viewing a profile on that page causes the tab to close. If that is the only tab open, the browser closes.

### Other extensions used

Bitwarden
Aria2 Integration
cookies.txt
Facebook Container

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

https://1drv.ms/v/c/cee37e748d67d3aa/Efcqs8SJUOJMuwWsh3yhTwcBG_n6RhPhlidSBWnYlTv9PA?e=rr7aEd

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.0
Firefox: 132
filterset (summary):
 network: 138383
 cosmetic: 49904
 scriptlet: 21370
 html: 2139
listset (total-discarded, last-updated):
 added:
  https://gist.githubusercontent.com/SmoochySix4/653ed595ee56af0a6d17e2401cce0ddc/raw/shrtswrd_filters.txt: 169-0, 5m
 default:
  user-filters: 0-0, never
  ublock-filters: 40910-136, 54m Δ
  ublock-badware: 11787-1, 54m Δ
  ublock-privacy: 1470-22, 54m Δ
  ublock-unbreak: 2565-1, 54m Δ
  ublock-quick-fixes: 212-10, 54m Δ
  easylist: 78953-204, 54m Δ
  easyprivacy: 53260-69, 54m Δ
  urlhaus-1: 20515-0, now
  plowe-0: 3537-996, now
filterset (user): [empty]
trustedset:
 added: [array of 3 redacted]
hostRuleset:
 added: [array of 2 redacted]
userSettings:
 cloudStorageEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 437 ms
 maxAssetCacheWait: 234 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 4
 network:
  adam4adam.com: 2
  googletagmanager.com: 1
  jsdelivr.net: 1
 extended:
  ##.remove-ads-link
  ##.header-banner
  ##.aside-right
  ##.promo-bar-ads
  ##.js-stop-ad-blocker-popup
  ##footer
```
</details>

**Comment by stephenhawk8054** [comment]
That's the expected behavior of `$popunder`

**Comment by SmoochySix4** [comment]
I don't think the expected behavior is to close the tab you are working in. Isn't in only supposed to stop from opening a popunder, but leave the original page alone?

**Comment by ghajini** [comment]
write a steps to reproduce, if this requires account, then we can't investogate

**Comment by SmoochySix4** [comment]
I provided an video of what was happening. I also checked all of the boxes, showing I followed the recommended/requested when reporting. Those are the steps to repeat the process. Also, you can create an account with a fake email account... in order to test.

**Comment by stephenhawk8054** [comment]
It's because the site redirects to new tab page when clicking on the logo button

![image](https://github.com/user-attachments/assets/82cfa4c7-3797-45fc-98df-78e11f5b6c9c)

As a result, the destination link is different than `https://www.adam4adam.com/` and `*$popunder` (without any other conditions) kicks in: https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#popunder

> To block "popunders" windows/tabs where the original page redirects to an advertisement and the desired content loads in the newly created one.

**Comment by stephenhawk8054** [comment]
As uAssets doesn't use the filter `*$popunder` (you are using custom filter lists), I'll convert into discussion.

**Comment by freezer2022** [comment]
Duplicate of | related to:
- https://github.com/uBlockOrigin/uAssets/discussions/24776

**Comment by SmoochySix4** [comment, marked as answer]
It appears that adding `_popunder_$badfilter` as a filter stops all of that from happening. With this and the `*$popunder`, it blocks popups and popunders without causing the tab to close.

**Comment by stephenhawk8054** [comment]
It's caused by the filter in your own custom list: https://gist.githubusercontent.com/SmoochySix4/653ed595ee56af0a6d17e2401cce0ddc/raw/shrtswrd_filters.txt

You just need to remove that filter.

**Reply by SmoochySix4** [reply]
What filter?

**Reply by stephenhawk8054** [reply]
https://gist.github.com/SmoochySix4/653ed595ee56af0a6d17e2401cce0ddc#file-shrtswrd_filters-txt-L5

**Reply by SmoochySix4** [reply]
I fixed it with what I marked as the answer. Thanks, anyways, for the help.
